content/en/docs/Getting Started/pipelines.md

```diff
@@ -1,191 +1,165 @@
+For this tutorial we are going to use [minikube][minikube] to run the commands
+locally.
 
-```bash
-kubectl apply -f task-goodbye.yaml
-```
+## Prerequisites
 
-To run this task with Tekton, you need to create a `TaskRun`, which is
-another Kubernetes object used to specify run time information for a `Task`. 
+- Complete the [Getting started with tasks](/docs/getting-started/tasks/)
+	tutorial. *Do not clean up your resources*, skip the last section.
+
+- [Install the Tekton CLI](/docs/cli/).
 
-To view this `TaskRun` object you can run the following Tekton CLI (`tkn`) command:
+## Creating and running a second task
 
-```shell
-tkn task start goodbye --dry-run
-```
+You already have a *Hello World!* task. To create a second *Goodbye World!*
+task:
 
-After running the command above, the following `TaskRun` definition should be shown:
+1.  Create a new file named  `goodbye-world.yaml` and add the following
+    content:
 
-```yaml
-apiVersion: tekton.dev/v1beta1
-kind: TaskRun
-metadata:
-  generateName: goodbye-run-
-spec:
-  taskRef:
-    name: goodbye
-```
+    ```yaml
+	  apiVersion: tekton.dev/v1beta1
+	  kind: Task
+	  metadata:
+	     name: goodbye
+	  spec:
+	     steps:
+	        - name: goodbye
+	          image: ubuntu
+	          script: |
+	             #!/usr/bin/env bash
+	             echo "Goodbye World!"
+    ```
 
-To use the `TaskRun` above to start the `echo` `Task`, you can either use 
-`tkn` or `kubectl`.
+1.  Apply your task file:
 
-Start with `tkn`:
+    ```bash
+    kubectl apply --filename goodbye-world.yaml
+    ```
 
-```shell
-tkn task start goodbye
-```
+When a task is part of a pipeline you don't have to instantiate it, the pipeline
+is going to take care of that.
 
-Start with `kubectl`:
+## Creating and running a pipeline
 
-```shell
-# use tkn's --dry-run option to save the TaskRun to a file
-tkn task start goodbye --dry-run > taskRun-goodbye.yaml
-# create the TaskRun
-kubectl create -f taskRun-goodbye.yaml
-```
+A **[pipeline](/docs/pipelines/pipelines/)** defines an ordered series of tasks
+arranged in a specific execution order as part of your CI/CD workflow.
 
-Tekton will now start running your `Task`. To see the logs of the `TaskRun`, run 
-the following `tkn` command:
+In this section you are going to create your first pipeline, that will include
+both the *Hello World!* and *Goodbye World!* tasks.
 
-```shell
-tkn taskrun logs --last -f 
-```
+1.  Create a new file named  `hello-goodbye-pipeline.yaml` and add the following
+    content:
 
-It may take a few moments before your `Task` completes. When it executes, it should 
-show the following output:
+    ```yaml
+    apiVersion: tekton.dev/v1beta1
+    kind: Pipeline
+    metadata:
+       name: hello-goodbye
+    spec:
+       tasks:
+          - name: hello
+            taskRef:
+               name: hello
+          - name: goodbye
+            runAfter:
+               - hello
+            TaskRef:
```

Comment:
should be `taskRef`

(i think this will work initially but it leads to really weird bugs X'D https://github.com/tektoncd/triggers/issues/526)

Reply:
Fixed!